[KO] Add Beatmap Nominators (#4961)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/ko.md

---
tags:
  - BN
  - BNG
---

# 비트맵 노미네이터

> *이 문서들도 확인해보세요: [비트맵 노미네이터 규칙](Rules), [비트맵 노미네이터 행동강령](Expectations), [비트맵 노미네이터를 위한 일반적인 정보](General_Information), 그리고 [비트맵 노미네이터가 되는 방법](Becoming_a_Beatmap_Nominator)*

Reviewer: ```suggestion
*이 문서들도 확인해보세요: [비트맵 노미네이터 규칙](Rules), [비트맵 노미네이터 행동지침](Expectations), [비트맵 노미네이터를 위한 일반적인 정보](General_Information), 그리고 [비트맵 노미네이터가 되는 방법](Becoming_a_Beatmap_Nominator)*
```
could be better; not sure

> **비트맵 노미네이터** (***BN***)는 [비트맵 지명](/wiki/Beatmap_ranking_procedure)을 담당하는 그룹입니다. 이들은 포럼과 인게임에서 보라색 이름을 가지고 있습니다.

Reviewer 2: in beatmap discussions 를 '인게임'으로 번역하셨는데, 홈페이지에 있는 '비트맵 토론 창'을 이야기하는 것이기 때문에 오역입니다.
```suggestion
**비트맵 노미네이터** (***BN***)는 [비트맵 지명](/wiki/Beatmap_ranking_procedure)을 담당하는 그룹입니다. 이들은 포럼과 비트맵 토론 창에서 보라색 이름으로 보여집니다.
```
'이들은 포럼과 비트맵 토론 창에서 보라색 이름으로 보여집니다.' 가 좀 더 자연스러운 느낌을 줄 수 있어보입니다.

## 역할과 책임

> 비트맵 노미네이터의 멤버들은 주로 [펜딩 비트맵](/wiki/Beatmap/Category#work-in-progress-and-pending)의 [모딩](/wiki/Modding) 및 랭킹 사이클에 올리는 것을 담당합니다. 비트맵 노미네이터는 매핑 또는 모딩과 관련된 조언을 합니다. 지명된 비트맵이 직접적으로 랭킹 기준을 위반하지 않더라도 수정해야 하는 문제가 있다고 생각되면 이들은 [비트맵 거부권](Beatmap_Veto)을 사용할 수 있습니다.

Reviewer 2: 굳이 멤버까지 직역할 필요는 없어보이며, '의'가 너무 많이 사용되어 번역투 느낌이 납니다. 그리고 이런 매핑 관련 용어들은 기존에 매퍼/모더들이 사용하던 용어 위주로 번역하는 게 좋고, 직역을 너무 많이 하게 되면 용어 사용에 있어 혼동이 올 수 있으니 추천드리지 않습니다.

```suggestion
BN은 주로 [보류 중 상태인 비트맵](/wiki/Beatmap/Category#work-in-progress-and-pending)을 [모딩](/wiki/Modding)하고 승인해 랭킹 사이클로 올리는 일을 담당합니다. 이들은 매핑이나 모딩에 관련된 내용들을 알려주거나 요청을 받습니다. 이미 지명받은 비트맵에 랭킹 크리테리아를 위반한 내용이 없으나 반드시 수정되어야 하는 문제가 있다고 생각하면 누구나 [비트맵 Veto](Beatmap_Veto)를 걸 수 있습니다.
```

> BN은 주로 [보류 중 상태인 비트맵](/wiki/Beatmap/Category#work-in-progress-and-pending)을 [모딩](/wiki/Modding)하고 승인해 랭킹 사이클로 올리는 일을 담당합니다. 이들은 매핑이나 모딩에 관련된 내용들을 알려주거나 요청을 받습니다. 이미 지명받은 비트맵에 랭킹 크리테리아를 위반한 내용이 없으나 반드시 수정되어야 하는 문제가 있다고 생각하면 누구나 [비트맵 Veto](Beatmap_Veto)를 걸 수 있습니다.

Reviewer: could be "Ranking criteria" or "랭크 기준" instead of "랭킹 크리테리아"
Currently It shows Ranking criteria on wiki
https://github.com/ppy/osu-wiki/blob/master/wiki/Ranking_Criteria/ko.md

> 추가로, 모든 비트맵 노미네이터들은 [비트맵 노미네이터에 대한 기대](Expectations)에 나온 대로 비트맵과 매퍼의 더 폭 넓은 관심을 유도하며 쉬지 않고 실력 향상을 위해 노력함으로써 더욱 큰 커뮤니티에 초점을 맞추는 것을 고수할 것으로 예상됩니다.

Reviewer 2: Expectation은 위에 지적한 내용과 동일합니다. '커뮤니티를 가리지 않고 다양한 맵과 매퍼들이 발전할 수 있도록 도움을 주어라' 가 메인 내용이므로 그 뜻을 확실히 표현하기 위해 일부 의역해 적어보았습니다.
```suggestion
추가로, 모든 BN은 [비트맵 노미네이터 행동강령](Expectations)에 명시되어 있듯이 다양한 맵과 매퍼들에게 관심을 가지고 그들의 실력 향상을 위해 노력함으로써 보다 큰 커뮤니티에 유연한 기준으로 도움을 주는 데에 초점을 둘 수 있도록 요구됩니다.
```

> 비트맵 노미네이터는 두 가지로 분류됩니다. 정식 노미네이터와 수습 노미네이터가 있습니다.

Reviewer 2: 비트맵 노미네이터라는 말 자체를 '노미네이터'만 분리해서 쓰는 경우는 거의 없으므로 좀 어색합니다. 
Full / Probation은 평소에 따로 번역해서 부르지 않았으나 편의상 번역본을 그대로 두고 추가로 영어를 적는 방향으로 하는게 좋아보입니다.
```suggestion
BN은 정식(Full)과 수습(Probation) 두 그룹으로 분류됩니다.
```

> ### 정식 비트맵 노미네이터

Reviewer 2: ```suggestion
### 정식 비트맵 노미네이터(Full BN)
```

> 정기적인 비트맵 모딩과 랭킹 외에도, 정식 비트맵 노미네이터들은 문제가 있다고 판명된 Qualified 섹션에 있는 비트맵을 실격시킬 책임이 있습니다. 또한 그들이 후에 NAT가 될 의향이 있을 경우 가끔씩 비트맵 노미네이터 신청 평가를 하기도 합니다.

Reviewer 2: Qualify / Disqualify는 평소 직역해서 부르지 않고 퀄파 / 디퀄이라 줄여 부르기 때문에 영어로 두는 걸 추천합니다. Full BN은 BN 사이트에서 신규 BN 평가하기를 활성화시켜 새로운 BN 신청 심사에 도움을 줄 수 있고, 이를 할 경우 NAT 팀에서 판단해서 신규 NAT이 필요할 때 그 동안 심사 평가에 도움을 준 BN 중 선발하는 것이기 때문에 이를 이해할 수 있도록 내용을 정리했습니다.
```suggestion
Full BN은 일반적인 비트맵 모딩과 랭크 외에도 Qualified 상태인 비트맵에 문제가 발견될 경우 Disqualify할 수 있는 권한을 가지고 있습니다. 이후 노미네이션 심사 팀(NAT)이 될 의향이 있다면 신규 BN 신청 심사에 참여할 수 있습니다.
```

> 약 3개월 마다 정식 비트맵 노미네이터는 평소 행동, 임명 및 그에 따른 퀄리티를 [NAT](/wiki/People/The_Team/Nomination_Assessment_Team)에서 평가받게 됩니다. 만약 그들이 용납될 수 없는 행동을 하거나, 눈에 띄게 지명을 자주 철회하는 경우 수습 비트맵 노미네이터로 강등될 수 있습니다.

Reviewer 2: 매핑 용어 수정.
```suggestion
Full BN은 대략 3개월마다 [NAT](/wiki/People/The_Team/Nomination_Assessment_Team)으로부터 행실과 지명 활동량/퀄리티를 평가받게 됩니다. 만약 지명한 맵들이 자주 중대한 이유로 리셋되거나 납득하기 어려운 행실을 보일 경우 Probation BN으로 강등당하게 됩니다.
```

> 최소한의 지명 활동을 하는 정식 비트맵 노미네이터들은 수습 강등 대상이 되지 않지만, 후에도 그들의 활동이 개선되지 않으면 그들은 그룹에서 제거됩니다.

Reviewer 2: below minimum -> 최소한이 아니라 최소 기준보다 낮게 한다는 말이므로 오역
```suggestion
최소 지명 활동량 기준보다 적게 할 경우 강등되지 않지만, 이후에도 계속 활동량을 늘리지 않는다면 BN 그룹에서 제거당하게 됩니다.
```

> ### 견습 비트맵 노미네이터

Reviewer 2: ```suggestion
### 견습 비트맵 노미네이터(Probation BN)
```

> 견습의 존재 이유는 신규, 그리고 관련 비트맵 노미네이터들을 보다 면밀하게 모니터링 하기 위함입니다. 수습 비트맵 노미네이터의 주된 역할은 랭킹 과정에서의 비트맵 단독 지명을 통해 노미네이터로서의 역할을 구축하는 것입니다. 최소 한 명의 정식 비트맵 노미네이터가 각 게임 모드의 랭킹 과정에 참여해야 하기에, 수습 노미네이터들은 자신의 게임 모드에 대하여 다른 수습 노미네이터가 지명한 비트맵을 최종 지명할 수 없습니다.

Reviewer 2: personal suggestion
```suggestion
Probation은 신규 BN이나 관련된 BN을 면밀하게 모니터링하기 위해 사용됩니다. Probation BN의 주 역할은 랭크 과정에 참여해 비트맵을 직접 지명해보면서 한 명의 BN이 되기 위한 기반을 다지는 것입니다.  랭크 과정 도중 맵셋에서 해당하는 게임 모드마다 최소 한 명의 Full BN이 필요하기 때문에, Probation BN은 이미 다른 Probation BN이 지명한 맵셋을 지명할 수 없습니다.
```

> 신규 비트맵 노미네이터는 1달의 수습 기간을 거치며, 그들의 지명 활동화 행동이 만족스러울 경우, 그들은 정식 비트맵 노미네이터로 승급하게 됩니다. 그렇지 않을 경우, 1달의 수습 기간을 추가로 거치거나 비트맵 노미네이터 자격을 잃게 됩니다.

Reviewer 2: ```suggestion
신규 BN은 1달의 수습 기간을 보냅니다. 기간동안 수행한 지명과 보여준 행실이 요건을 충족했을 경우 Full BN으로 승급하게 됩니다. 그렇지 않을 경우, 추가로 1달의 수습 기간을 보내거나 BN 자격을 잃게 됩니다.
```

> 비트맵 노미네이터가 수습 노미네이터가 될 경우, 그들은 같은 이유로 수습 노미네이터가 될 수 없습니다. 예를 들면, 만약 비트맵 노미네이터가 불성실한 행동으로 수습 노미네이터가 되었다면, 그가 불성실한 행동을 다시 할 경우 정식 비트맵 노미네이터라 해도 노미네이터 그룹에서 제거됩니다.

Reviewer 2: ```suggestion
어떤 이유로 인해 Probation BN으로 강등당할 경우, 동일한 이유로 다시 Probation BN으로 강등당할 수 없습니다. 예를 들어 어떤 BN이 미숙한 행실로 인해 Probation BN으로 강등당한 적이 있다면, 이후 미숙한 행실을 다시 저지를 경우 Full BN이라 해도 Probation BN으로 강등되지 않고 바로 BN 자격을 잃게 됩니다. 
```

> ## 팀 멤버들

Reviewer 2: 개인적으로 복수형을 그대로 번역하는 거 보단 단수형으로 번역하는 게 더 깔끔해보입니다.
```suggestion
## 팀 멤버
```

> 모든 [비트맵 노미네이터](https://osu.ppy.sh/groups/28)와 [비트맵 노미네이터 (수습)](https://osu.ppy.sh/groups/32)의 목록 입니다.

Reviewer 2: 해당 링크를 통해 그룹 페이지에서 멤버 목록을 확인할 수 있다는 내용 명확히 표기했습니다.
```suggestion
모든 [비트맵 노미네이터](https://osu.ppy.sh/groups/28)와 [비트맵 노미네이터 (Probation)](https://osu.ppy.sh/groups/32) 멤버 목록을 해당 그룹 페이지에서 확인할 수 있습니다.
```

> 아래 나열된 표는 각 비트맵 노미네이터가 공식적으로 지명할 수 있는 게임 모드를 나타냅니다. *참고: 모든 비트맵 노미네이터들은 아래 추가로 명시된 언어 이외에는 영어를 사용합니다.*

Reviewer 2: Additional languages에 대한 통일 필요 // '추가 언어' 추천
```suggestion
아래 표에서 각 BN이 공식적으로 지명할 수 있는 게임 모드를 확인할 수 있습니다. *참고: 모든 BN은 영어를 구사하며 별개로 명시된 추가 언어를 구사할 수 있습니다.*
```

> ### 모든 비트맵 

Reviewer 2: 오역
```suggestion
### Full BN
```

#### osu!standard

> | 닉네임 | 추가적으로 가능한 언어 |

Reviewer 2: ```suggestion
| 닉네임 | 추가 언어 |
```

| :-- | :-- |
| ![][flag_PH] [-Aqua](https://osu.ppy.sh/users/7150015) | 필리핀어 |
| ![][flag_ID] [-Keitaro](https://osu.ppy.sh/users/3378391) | 인도네시아어 |

> | ![][flag_CA] [Agatsu](https://osu.ppy.sh/users/5579871) |  |

Reviewer 2: ```suggestion
```

| ![][flag_TW] [bossandy](https://osu.ppy.sh/users/360437) | 중국어 |
| ![][flag_GB] [C00L](https://osu.ppy.sh/users/4930630) | 폴란드어 |
| ![][flag_US] [Cheri](https://osu.ppy.sh/users/5226970) |  |
| ![][flag_CL] [Cris-](https://osu.ppy.sh/users/6175280) | 스페인어 |
| ![][flag_CL] [Crissa](https://osu.ppy.sh/users/5405836) | 스페인어 |
| ![][flag_GB] [DeviousPanda](https://osu.ppy.sh/users/4966334) |  |
| ![][flag_BE] [Een Persoon](https://osu.ppy.sh/users/10959501) | 네덜란드어 |
| ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410) | 스페인어 |
| ![][flag_CA] [Elayue](https://osu.ppy.sh/users/6400861) |  |
| ![][flag_US] [fieryrage](https://osu.ppy.sh/users/3533958) |  |
| ![][flag_CN] [Firika](https://osu.ppy.sh/users/9590557) | 중국어 |
| ![][flag_CN] [Garden](https://osu.ppy.sh/users/2849992) | 중국어 |
| ![][flag_ID] [Hinsvar](https://osu.ppy.sh/users/1249323) | 인도네시아어 |
| ![][flag_US] [Hobbes2](https://osu.ppy.sh/users/8157492) | 텔루구어 |
| ![][flag_CA] [jonathanlfj](https://osu.ppy.sh/users/270377) | 중국어, 약간의 프랑스어 |
| ![][flag_CN] [Karen](https://osu.ppy.sh/users/3143784) | 중국어 |
| ![][flag_CL] [Krisom](https://osu.ppy.sh/users/99269) | 스페인어 |

> | ![][flag_CA] [Lafayla](https://osu.ppy.sh/users/5312547) |  |

Reviewer 2: ```suggestion
| ![][flag_DE] [Lasse](https://osu.ppy.sh/users/896613) | 독일어 |
```

| ![][flag_GB] [Log Off Now](https://osu.ppy.sh/users/4378277) |  |

> | ![][flag_KR] [Luscent](https://osu.ppy.sh/users/2688581) | 한국어 |

Reviewer 2: ```suggestion
```

| ![][flag_CN] [Mafumafu](https://osu.ppy.sh/users/3076909) | 중국어 |
| ![][flag_RS] [MaridiuS](https://osu.ppy.sh/users/4496961) | 세르비아어 |
| ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) | 스페인어 |
| ![][flag_US] [Mimari](https://osu.ppy.sh/users/14339830) |  |
| ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009) | 러시아어 |

> | ![][flag_DE] [Mordred](https://osu.ppy.sh/users/7265097) | 독일어 |

Reviewer 2: ```suggestion
| ![][flag_DE] [Mordred](https://osu.ppy.sh/users/7265097) | 독일어 |
| ![][flag_DK] [Morrighan](https://osu.ppy.sh/users/12042090) | 덴마크어 |
```

| ![][flag_US] [Nana Abe](https://osu.ppy.sh/users/6443117) |  |

> | ![][flag_US] [Nao Tomori](https://osu.ppy.sh/users/5364763) |  |

Reviewer 2: ```suggestion
```

| ![][flag_SG] [neonat](https://osu.ppy.sh/users/1561995) | 중국어 |
| ![][flag_US] [Nevo](https://osu.ppy.sh/users/7451883) |  |
| ![][flag_GB] [NexusQI](https://osu.ppy.sh/users/13822800) |  |
| ![][flag_FR] [Nozhomi](https://osu.ppy.sh/users/2716981) | 프랑스어 |
| ![][flag_IT] [Nuvolina](https://osu.ppy.sh/users/10974170) | 이탈리아어 |
| ![][flag_AU] [Pentori](https://osu.ppy.sh/users/7452237) |  |
| ![][flag_HK] [Petal](https://osu.ppy.sh/users/7354729) | 중국어, 광둥어 |
| ![][flag_BR] [pimp](https://osu.ppy.sh/users/328615) | 포르투갈어 |
| ![][flag_FR] [Realazy](https://osu.ppy.sh/users/918297) | 프랑스어 |
| ![][flag_KR] [Riana](https://osu.ppy.sh/users/1997633) | 한국어 |
| ![][flag_RU] [riffy](https://osu.ppy.sh/users/597957) | 러시아어 |

> | ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231) | 러시아어 |

Reviewer 2: ```suggestion
| ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231) | 러시아어 |
| ![][flag_GB] [skylaa](https://osu.ppy.sh/users/9505704) |  |
```

| ![][flag_RU] [SMOKELIND](https://osu.ppy.sh/users/9327302) | 러시아어 |
| ![][flag_CA] [Sparhten](https://osu.ppy.sh/users/7601720) |  |

> | ![][flag_GB] [spoes](https://osu.ppy.sh/users/9505704) |  |

Reviewer 2: ```suggestion
```

> | ![][flag_BE] [Stack](https://osu.ppy.sh/users/6122935) | 네덜란드어 |

Reviewer 2: ```suggestion
```

| ![][flag_US] [StarCastler](https://osu.ppy.sh/users/12402453) |  |
| ![][flag_AT] [Stixy](https://osu.ppy.sh/users/9000308) | 독일어, 세르비아어 |
| ![][flag_DE] [Sylvarus](https://osu.ppy.sh/users/4505918) | 독일어 |
| ![][flag_EG] [Teky](https://osu.ppy.sh/users/10520912) | 아랍어 |
| ![][flag_FI] [TheKingHenry](https://osu.ppy.sh/users/5128277) | 핀란드어 |

> | ![][flag_TH] [timemon](https://osu.ppy.sh/users/2072005) | 타이어 |

Reviewer 2: ```suggestion
| ![][flag_TH] [timemon](https://osu.ppy.sh/users/2072005) | 태국어 |
```

| ![][flag_US] [UberFazz](https://osu.ppy.sh/users/8646059) |  |
| ![][flag_CL] [Uberzolik](https://osu.ppy.sh/users/1314547) | 스페인어, 프랑스어 |

> | ![][flag_PL] [Zelq](https://osu.ppy.sh/users/8953955) | Polish |

Reviewer 2: ```suggestion
| ![][flag_PL] [Zelq](https://osu.ppy.sh/users/8953955) | 폴란드어 |
```

> | ![][flag_PL] [Zelq](https://osu.ppy.sh/users/8953955) | 폴란드어 |

Reviewer 2: ```suggestion
| ![][flag_SE] [Zer0-](https://osu.ppy.sh/users/4260033) | 스웨덴어 |
```

#### osu!taiko

> | 닉네임 | 추가적으로 가능한 언어 |

Reviewer 2: ```suggestion
| 닉네임 | 추가 언어 |
```

| :-- | :-- |
| ![][flag_GB] [\_DUSK\_](https://osu.ppy.sh/users/6092181) | 우르두어, 약간의 아랍어 |
| ![][flag_DE] [Capu](https://osu.ppy.sh/users/2474015) | 독일어 |
| ![][flag_US] [Cychloryn](https://osu.ppy.sh/users/6921736) | 약간의 일본어 |
| ![][flag_HK] [Cynplytholowazy](https://osu.ppy.sh/users/3901754) | 중국어, 광둥어 |
| ![][flag_VN] [davidminh0111](https://osu.ppy.sh/users/9623142) | 베트남어 |
| ![][flag_PR] [Default Guy](https://osu.ppy.sh/users/7058457) | 스페인어 |
| ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410) | 스페인어 |
| ![][flag_PH] [Eyenine](https://osu.ppy.sh/users/1259391) | 필리핀어 |
| ![][flag_TR] [frukoyurdakul](https://osu.ppy.sh/users/7612550) | 터키어 |

> | ![][flag_EC] [Gamelan4](https://osu.ppy.sh/users/9856910) | 스페인어 |

Reviewer 2: ```suggestion
| ![][flag_EC] [Gamelan4](https://osu.ppy.sh/users/9856910) | 스페인어 |
| ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) | 아랍어, 프랑스어, 약간의 이탈리아어 |
```

| ![][flag_NL] [ikin5050](https://osu.ppy.sh/users/4007649) | 네덜란드어 |
| ![][flag_FI] [Jaltzu](https://osu.ppy.sh/users/2597417) | 핀란드어, 일본어 |
| ![][flag_MY] [Jerry](https://osu.ppy.sh/users/605973) |  |
| ![][flag_HK] [kanpakyin](https://osu.ppy.sh/users/394326) | 일본어, 광둥어, 중국어 |

> | ![][flag_US] [Nao Tomori](https://osu.ppy.sh/users/5364763) |  |

Reviewer 2: ```suggestion
```

> | ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | 광둥어, 중국어 |

Reviewer 2: 순서 오류
```suggestion
| ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | 중국어, 광둥어 |
```

| ![][flag_SG] [neonat](https://osu.ppy.sh/users/1561995) | 중국어 |
| ![][flag_US] [radar](https://osu.ppy.sh/users/7131099) |  |
| ![][flag_FR] [realy0\_](https://osu.ppy.sh/users/8863359) | 프랑스어 |

> | ![][flag_AT] [Stefan](https://osu.ppy.sh/users/626907) | 독일어, 세르비아어 |

Reviewer 2: ```suggestion
| ![][flag_AT] [Stefan](https://osu.ppy.sh/users/626907) | 독일어, 세르비아어 |
| ![][flag_CL] [Ulqui](https://osu.ppy.sh/users/1263669) | 스페인어 |
```

#### osu!catch

| 닉네임 | 추가적으로 가능한 언어 |

> | :-- | :-- |

Reviewer 2: ```suggestion
| :-- | :-- |
| ![][flag_HK] [autofanboy](https://osu.ppy.sh/users/636114) | 중국어, 광둥어 |
```

| ![][flag_CL] [Bastian](https://osu.ppy.sh/users/6345176) | 스페인어 |
| ![][flag_NO] [Benita](https://osu.ppy.sh/users/4023183) | 노르웨이어 |

> | ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134) | 필리핀어 |

Reviewer 2: ```suggestion
| ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134) | 필리핀어 |
| ![][flag_DE] [Myxo](https://osu.ppy.sh/users/2202645) | 독일어 |
```

| ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164) | 러시아어 |
| ![][flag_GR] [Nokashi](https://osu.ppy.sh/users/5431196) | 그리스어 |
| ![][flag_PL] [Phob](https://osu.ppy.sh/users/6069462) | 폴란드어 |

> | ![][flag_KR] [Rocma](https://osu.ppy.sh/users/566276) | 한국어 |

Reviewer 2: ```suggestion
```

| ![][flag_US] [Secre](https://osu.ppy.sh/users/2306637) |  |

#### osu!mania

| 닉네임 | 추가적으로 가능한 언어 |
| :-- | :-- |
| ![][flag_CN] [\_Stan](https://osu.ppy.sh/users/1653229) | 중국어 |
| ![][flag_ID] [Amiichii](https://osu.ppy.sh/users/9228569) | 인도네시아어 |
| ![][flag_ID] [Arzenvald](https://osu.ppy.sh/users/3027421) | 인도네시아어 |
| ![][flag_SE] [Davvy](https://osu.ppy.sh/users/10047413) | 스웨덴어 |
| ![][flag_KR] [Dubstek](https://osu.ppy.sh/users/9555243) | 한국어 |

> | ![][flag_KR] [Fresh Chicken](https://osu.ppy.sh/users/3984370) | 한국어, 일본어 |

Reviewer 2: ```suggestion
| ![][flag_KR] [Fresh Chicken](https://osu.ppy.sh/users/3984370) | 한국어, 일본어 |
| ![][flag_CA] [guden](https://osu.ppy.sh/users/11626065) |  |
```

| ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573) | 필리핀어 |
| ![][flag_ID] [Lirai](https://osu.ppy.sh/users/3734394) | 인도네시아어 |

> | ![][flag_KR] [Murumoo](https://osu.ppy.sh/users/8001433) | 한국어 |

Reviewer 2: ```suggestion
| ![][flag_KR] [Murumoo](https://osu.ppy.sh/users/8001433) | 한국어 |
| ![][flag_HK] [PokeSky](https://osu.ppy.sh/users/3617111) | 중국어, 광둥어 |
```

| ![][flag_ID] [RiP46](https://osu.ppy.sh/users/5222596) | 인도네시아어 |
| ![][flag_NL] [Roasted Chicken](https://osu.ppy.sh/users/9461160) | 네덜란드어 |
| ![][flag_CN] [Shima Rin](https://osu.ppy.sh/users/6089608) | 중국어 |
| ![][flag_ID] [Stelar](https://osu.ppy.sh/users/15205070) | 인도네시아어 |
| ![][flag_KR] [Sun](https://osu.ppy.sh/users/4115819) | 한국어 |
| ![][flag_US] [Unpredictable](https://osu.ppy.sh/users/7560872) |  |

> ### 견습 비트맵 노미네이터들

Reviewer 2: ```suggestion
### Probation BN
```

#### osu!standard

| 닉네임 | 추가적으로 가능한 언어 |

> | :-- | :-- |

Reviewer 2: ```suggestion
| :-- | :-- |
| ![][flag_GB] [AJT](https://osu.ppy.sh/users/3181083) |  |
| ![][flag_US] [Battle](https://osu.ppy.sh/users/4037545) |  |
```

> | ![][flag_CN] [Dored](https://osu.ppy.sh/users/10284894) | 중국어 |

Reviewer 2: ```suggestion
| ![][flag_CN] [Dored](https://osu.ppy.sh/users/10284894) | 중국어 |
| ![][flag_KR] [Luscent](https://osu.ppy.sh/users/2688581) | 한국어 |
```

> | ![][flag_DK] [Morrighan](https://osu.ppy.sh/users/12042090) | 덴마크어 |

Reviewer 2: ```suggestion
| ![][flag_DE] [Mir](https://osu.ppy.sh/users/8688812) | 독일어 |
| ![][flag_RU] [NeKroMan4ik](https://osu.ppy.sh/users/11387664) | 러시아어 |
```

| ![][flag_FR] [realy0\_](https://osu.ppy.sh/users/8863359) | 프랑스어 |
| ![][flag_US] [rosario wknd](https://osu.ppy.sh/users/6341518) |  |

> | ![][flag_KR] [Sonnyc](https://osu.ppy.sh/users/11771) | 한국어 |

Reviewer 2: ```suggestion
| ![][flag_KR] [Sonnyc](https://osu.ppy.sh/users/11771) | 한국어 |
| ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | 약간의 영어 |
```

| ![][flag_FR] [Xahlt](https://osu.ppy.sh/users/13466351) | 프랑스어 |
| ![][flag_CN] [Yugu](https://osu.ppy.sh/users/3161834) | 중국어, 일본어 |

> | ![][flag_SE] [Zer0-](https://osu.ppy.sh/users/4260033) | 스웨덴어 |

Reviewer 2: ```suggestion
```

#### osu!taiko

| 닉네임 | 추가적으로 가능한 언어 |
| :-- | :-- |

> | ![][flag_TN] [Hivie](https://osu.ppy.sh/users/14102976) | 아랍어, 프랑스어, 약간의 이탈리아어 |

Reviewer 2: ```suggestion
```

> | ![][flag_CL] [Ulqui](https://osu.ppy.sh/users/1263669) | 스페인어 |

Reviewer 2: ```suggestion
| ![][flag_FI] [gevbiivi5](https://osu.ppy.sh/users/7140680) | 약간의 일본어, 약간의 독일어 |
| ![][flag_GB] [Horiiizon](https://osu.ppy.sh/users/8071438) |  |
| ![][flag_AT] [Yasuho](https://osu.ppy.sh/users/8458835) | 독일어, 약간의 프랑스어 |
```

#### osu!catch

| 닉네임 | 추가적으로 가능한 언어 |
| :-- | :-- |

> | ![][flag_HK] [autofanboy](https://osu.ppy.sh/users/636114) | 중국어, 광둥어 |

Reviewer 2: ```suggestion
```

> | ![][flag_NL] [GIGACHAD](https://osu.ppy.sh/users/11081858) | 네덜란드어 |

Reviewer 2: ```suggestion
```

> | ![][flag_DE] [Myxo](https://osu.ppy.sh/users/2202645) | 독일어 |

Reviewer 2: ```suggestion
| ![][flag_ID] [Xinely](https://osu.ppy.sh/users/1521445) | 인도네시아어 |
```

#### osu!mania

| 닉네임 | 추가적으로 가능한 언어 |
| :-- | :-- |
| ![][flag_TW] [deta5859](https://osu.ppy.sh/users/15275662) | 중국어 |

> | ![][flag_CA] [guden](https://osu.ppy.sh/users/11626065) |  |

Reviewer 2: ```suggestion
```

| ![][flag_ID] [Mipha-](https://osu.ppy.sh/users/5767941) | 인도네시아어 |

> | ![][flag_HK] [PokeSky](https://osu.ppy.sh/users/3617111) | 중국어, 광둥어 |

Reviewer 2: ```suggestion
```

## 역사

> - 비트맵 노미네이터는 예전에는 *비트맵 감사팀* (BAT)로 불렸습니다.

Reviewer 2: 과도한 '는' 사용
```suggestion
- BN은 예전에 *비트맵 감사 팀* (Beatmap Appreciation Team, BAT)으로 불렸습니다.
```

> - 2014년 8월 22일 부터, BAT는 더 이상 스태프의 일원이 아니게 되었습니다. 이로 인해 인게임 채팅 관리 권한 및 인게임과 포럼에서의 빨간 닉네임이 사라지게 되었습니다. 

Reviewer 2: ```suggestion
- 2014년 8월 22일부로  BAT는 스태프 팀에서 제외되었고, 이로 인해 인게임 채팅 관리 권한 및 인게임과 포럼에서 빨간 닉네임이 사라지게 되었습니다. 
```

> - 2015년 2월 1일, *비트맵 감사팀*에서 *비트맵 노미네이션 그룹*으로 변경되었고, 후에 우리가 아는 비트맵 노미네이터로 바뀌게 됩니다.

Reviewer 2: ```suggestion
- 2015년 2월 1일부로 *BAT*에서 *비트맵 노미네이션 그룹(Beatmap Nomination Group)*으로 변경되었고, 이후 우리가 현재 알고 있는 비트맵 노미네이터로 변경되었습니다.
```

[flag_AT]: /wiki/shared/flag/AT.gif "오스트리아"
[flag_AU]: /wiki/shared/flag/AU.gif "호주"
[flag_BE]: /wiki/shared/flag/BE.gif "벨기에"
[flag_BR]: /wiki/shared/flag/BR.gif "브라질"
[flag_BY]: /wiki/shared/flag/BY.gif "벨라루스"
[flag_CA]: /wiki/shared/flag/CA.gif "캐나다"
[flag_CL]: /wiki/shared/flag/CL.gif "칠레"
[flag_CN]: /wiki/shared/flag/CN.gif "중국"
[flag_DE]: /wiki/shared/flag/DE.gif "독일"
[flag_DK]: /wiki/shared/flag/DK.gif "덴마크"
[flag_EC]: /wiki/shared/flag/EC.gif "에콰도르"
[flag_EG]: /wiki/shared/flag/EG.gif "이집트"
[flag_FI]: /wiki/shared/flag/FI.gif "핀란드"
[flag_FR]: /wiki/shared/flag/FR.gif "프랑스"
[flag_GB]: /wiki/shared/flag/GB.gif "영국"
[flag_GR]: /wiki/shared/flag/GR.gif "그리스"
[flag_HK]: /wiki/shared/flag/HK.gif "홍콩"
[flag_ID]: /wiki/shared/flag/ID.gif "인도네시아"
[flag_IT]: /wiki/shared/flag/IT.gif "이탈리아"
[flag_JP]: /wiki/shared/flag/JP.gif "일본"
[flag_KR]: /wiki/shared/flag/KR.gif "대한민국"
[flag_MY]: /wiki/shared/flag/MY.gif "말레이시아"
[flag_NL]: /wiki/shared/flag/NL.gif "네덜란드"
[flag_NO]: /wiki/shared/flag/NO.gif "노르웨이"
[flag_NZ]: /wiki/shared/flag/NZ.gif "뉴질랜드"
[flag_PH]: /wiki/shared/flag/PH.gif "필리핀"

> [flag_PL]: /wiki/shared/flag/PL.gif "폴란트"

Reviewer 2: ```suggestion
[flag_PL]: /wiki/shared/flag/PL.gif "폴란드"
```

[flag_PR]: /wiki/shared/flag/PR.gif "푸에르토리코"
[flag_RS]: /wiki/shared/flag/RS.gif "세르비아"
[flag_RU]: /wiki/shared/flag/RU.gif "러시아"
[flag_SE]: /wiki/shared/flag/SE.gif "스웨덴"
[flag_SG]: /wiki/shared/flag/SG.gif "싱가포르"
[flag_TH]: /wiki/shared/flag/TH.gif "태국"
[flag_TN]: /wiki/shared/flag/TN.gif "튀니지"
[flag_TR]: /wiki/shared/flag/TR.gif "터키"

> [flag_TW]: /wiki/shared/flag/TW.gif "타이완"

Reviewer 2: ```suggestion
[flag_TW]: /wiki/shared/flag/TW.gif "대만"
```

[flag_US]: /wiki/shared/flag/US.gif "미국"

> [flag_VN]: /wiki/shared/flag/VN.gif ""

Reviewer 2: ```suggestion
[flag_VN]: /wiki/shared/flag/VN.gif "베트남"
```
